pkgs/tools/misc/flashrom-stable/default.nix

```
@@ -0,0 +1,36 @@
+{ fetchgit
+, installShellFiles
+, lib
+, libftdi1
+, libgpiod
+, libjaylink
+, libusb1
+, pciutils
+, pkg-config
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flashrom-stable";
+  version = "1.1";
+
+  src = fetchgit {
+    url = "https://review.coreboot.org/flashrom-stable";
+    rev = "272aae888ce5abf5e999d750ee4577407db32246";
+    sha256 = "sha256-DR4PAING69+TMsyycGxt1cu0ba1tTlG36+H/pJ0oP4E=";
```

```suggestion
    hash = "sha256-DR4PAING69+TMsyycGxt1cu0ba1tTlG36+H/pJ0oP4E=";
```

Since you are using an SRI hash, we don't have to rely on the `sha256` attribute.